packages/lib/src/components/internal/FormFields/Checkbox/Checkbox.scss

```diff
@@ -7,7 +7,6 @@
         position: relative;
         padding-left: 24px;
         cursor: pointer;
-        display: inline-block;
```

Comment:
I tested and this causes a change in OpenInvoice component.

Reply:
Can you highlight what is the change? Looking at the Open Invoices page in the old playground, I can't spot any problem

Reply:
https://github.com/Adyen/adyen-web/assets/6471825/37125f4b-f1c2-492c-974c-9319e8da5325

The top margin shifts but also the [ ] checkbox square itself shifts by 1px.

Reply:
The checkbox shift is fine I believe, as the text gets a better vertical alignment next to the Checkbox. Regarding the margin shift, I will take a look with Arjen what is the correct pixels there

Reply:
I updated some styles - the CSS should be ok now. Margin between elements is 16px, and from the pay button is 24px.